packages/webdriverio/src/test.ts

```diff
@@ -63,14 +62,24 @@ describe('@axe-core/webdriverio', () => {
     let server: Server;
     let addr: string;
     let client: webdriverio.BrowserObject;
-    const axeSourcePath = path.resolve(
-      './fixtures/external/axe-core@legacy.js'
+    const axePath = require.resolve('axe-core');
+    const axeSource = fs.readFileSync(axePath, 'utf8');
+    const axeTestFixtures = path.resolve(
+      __dirname,
+      '..',
+      'fixtures',
+      'external'
```

Comment:
I'm not sure how this is passing CI, but I don't see the fixture files in package.json. Am I missing something?

Reply:
They're a root-level dependency, see https://github.com/dequelabs/axe-core-npm/blob/develop/package.json#L49